index.html

```diff
@@ -996,7 +996,7 @@ <h1>Speakers</h1>
 											<a href="https://kushaldas.in"><i class="icon fa fa-external-link"></i></a>
 											<a href="http://github.com/kushaldas"><i class="icon fa fa-github"></i></a>
 											<a href="https://twitter.com/kushaldas"><i class="icon social_twitter"></i></a>
-											<a href="#"><i class="icon social_linkedin"></i></a>
+											<a href="https://www.linkedin.com/company/7900744/"><i class="icon social_linkedin"></i></a>
```

Comment:
I believe this is the linkedin profile: https://www.linkedin.com/in/kushal-das-5a655b8/